pages/pricing.tsx

```diff
@@ -277,6 +276,26 @@ export default function PricingPage() {
                     />
                   </td>
                 </tr>
+                <tr>
+                  <td style={{ textAlign: "left" }}>SaaS Hosted</td>
```

Comment:
@andyjih is using the word "SaaS" OK?  I thought we wanted to use "Hosed by Grouparoo" or something like that?

Reply:
@evantahler I think SaaS feels right to me here. Just using the word "Hosted" suggests they're only paying for hosting, which isn't entirely true.